_posts/2020-02-01-fosdem-2020.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title:  Open Source Design speak at FOSDEM 2020
+date:   2020-02-01
+categories:
+  - open-source
+  - communities
+  - design
+  - conferences
+  - fosdem
+authors:
+  - amitlzkpa
+---
+
+
+During the first weekend of February 2020 Open Source Design(OSD) hosted the 6th edition of our devroom at FOSDEM.
```

Comment:
```suggestion
During the first weekend of February 2020 Open Source Design (OSD) hosted the 6th edition of our devroom at FOSDEM.
```
almost missed this typo!

Reply:
Intersting...I used to think the space before the brackets are part of the writer's "creative freedom"
Something new to learn everyday